lib/src/sdk.ts

```diff
@@ -201,6 +238,12 @@ export class ExchangeSDK {
       family: currency.family,
       amount,
       recipient,
+      maxFeePerGas,
```

Comment:
It is strange that you can add all those parameters directly without checking the `family` as they seems to be ethereum-only parameters.

For instance, this will not work for a cosmos transaction :/